OPUNetTransportLayer.cpp

```diff
@@ -664,9 +663,9 @@ int OPUNetTransportLayer::Receive(Packet& packet)
 
 
 // **DEBUG**
-//logFile << "ReadSocket: type = " << (int)packet.header.type
-//		<< "  commandType = " << packet.tlMessage.tlHeader.commandType
-//		<< "  sourcePlayerNetID = " << packet.header.sourcePlayerNetID << endl;
+//Log("ReadSocket: type = " + std::to_string(static_cast<int>(packet.header.type))

```

Comment:
I don't know if the `static_cast` is needed here. The `<<` operator would actually write out a character when passed a `char` type. I imagine the `to_string` method would just up cast to an `int` and always assume a numeric type.

Reply:
I wasn't sure of behaviour so thought it was better to just be explicit. I figure others might question it as well.

If you would prefer, I can test it out and remove the static_cast if not needed.

Reply:
Yeah, I think the `static_cast` should go. It'll end up just raising questions and confusing people. Probably good that it'll be removed it in it's own commit.